modules/stitching/src/blenders.cpp

```diff
@@ -248,6 +249,15 @@ void MultiBandBlender::prepare(Rect dst_roi)
 #if defined(HAVE_OPENCV_CUDAARITHM) && defined(HAVE_OPENCV_CUDAWARPING)
     if (can_use_gpu_)
     {
+        initialized = false;
+        feed_idx = 0;
+
+        tl_points_ = std::vector<Point>();
+        gpu_weight_pyr_gauss_vec_ = std::vector<std::vector<cuda::GpuMat>>();
+        gpu_src_pyr_laplace_vec_ = std::vector<std::vector<cuda::GpuMat>>();
+        ups_ = std::vector<std::vector<cuda::GpuMat>>();
+        imgs_with_border_ = std::vector<cuda::GpuMat>();
```

Comment:
Why we need these "empty" assignments? Consider using `.clear()` instead.

Reply:
No reason I think. Changed to clear.